### PR TITLE
chore: bump emailjs & @types/node

### PR DESCRIPTION
**Pull request by dammy001**

### CODE REVIEW

**Comment by dammy001** [comment]
@scopsy here is the PR. I also update the @types/node to match the node versions

**Review by p-fernandez — CHANGES_REQUESTED** [review]
Please would you mind to address the changes requested?
Also you upgraded the `@types/node` version, you should revert back to the original as the Node v18 update is not done in our side.
Also the target back to `es2017`.

**Comment by dammy001** [comment]
> Please would you mind to address the changes requested? Also you upgraded the `@types/node` version, you should revert back to the original as the Node v18 update is not done in our side. Also the target back to `es2017`.

@p-fernandez Since we're updating the node version, I think it's better to change the transpile target which should result in a smaller library size, unlocking lib features, and sometimes even (slightly) better performance. check here: https://github.com/microsoft/TypeScript/wiki/Node-Target-Mapping for node target mapping

**Comment by p-fernandez** [comment]
> > Please would you mind to address the changes requested? Also you upgraded the `@types/node` version, you should revert back to the original as the Node v18 update is not done in our side. Also the target back to `es2017`.
> 
> @p-fernandez Since we're updating the node version, I think it's better to change the transpile target which should result in a smaller library size, unlocking lib features, and sometimes even (slightly) better performance. check here: https://github.com/microsoft/TypeScript/wiki/Node-Target-Mapping for node target mapping
> 
> I can revert to `ES2021` instead of `ESNext`. what do you think?
> 
> For the `@types/node` I think we should update it to match the node version.

I do agree but needs to be worked in the PR where the Node is upgraded so all the changes are together. Both of the changes I asked to revert. 🙏🏻

File: apps/web/package.json [hunk 1]
```diff
@@ -135,8 +130,12 @@
     "@storybook/react": "^7.4.2",
     "@storybook/react-webpack5": "^7.4.2",
     "@testing-library/jest-dom": "^4.2.4",
-    "@types/react": "^17.0.0",
     "@types/testing-library__jest-dom": "^5.14.5",
+    "@types/jest": "^29.5.0",
+    "@types/node": "^18.0.0 || >=20.0.0",
+    "@types/react": "^17.0.0",
+    "@types/react-dom": "^17.0.0",
+    "@types/react-table": "^7.7.12",
```
[the remark this answers is not in the document]
Comment (dammy001) [reply apps/web/package.json:138]: They are `devDependencies` that don't have to be part of the build. Adding the type definition to dependencies will make the dependency tree bloated.

**Comment by dammy001** [comment]
> > > Please would you mind to address the changes requested? Also you upgraded the `@types/node` version, you should revert back to the original as the Node v18 update is not done in our side. Also the target back to `es2017`.
> > 
> > 
> > @p-fernandez Since we're updating the node version, I think it's better to change the transpile target which should result in a smaller library size, unlocking lib features, and sometimes even (slightly) better performance. check here: https://github.com/microsoft/TypeScript/wiki/Node-Target-Mapping for node target mapping
> > I can revert to `ES2021` instead of `ESNext`. what do you think?
> > For the `@types/node` I think we should update it to match the node version.
> 
> I do agree but needs to be worked in the PR where the Node is upgraded so all the changes are together. Both of the changes I asked to revert. 🙏🏻

@p-fernandez i get but i think this PR is pointing to the `node-upgrade` branch as advised by @scopsy

File: apps/web/package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (p-fernandez) [reply apps/web/package.json:138]: Sorry, you are right. I missed that it was moved from dependency place.
Please keep this change!

**Pull request closed without merging**